sp7-stats/index.php

```
@@ -166,223 +166,258 @@ class="form-control"><?=$result_2?></select>
 
-			echo '<li>' . urldecode($discipline) . '<ul>';
+                        echo '<li>' . urldecode($discipline) . '<ul>';
 
-			foreach($collections as $collection => $data){
+                        foreach($collections as $collection => $data){
 
-				echo '<li data-reports_count="'.$data['count'].'">
-					<a href="'.LINK.'institution/?institution='.$institution.'&discipline='.$discipline.'&collection='.$collection.'&year='.$year.'&month='.$month.'">'.urldecode($collection).'</a> ['.$data['count'].']
-					<br>Specify 7 versions:<ul>';
+                                echo '<li data-reports_count="'.$data['count'].'">
+                                        <a href="'.LINK.'institution/?institution='.$institution.'&discipline='.$discipline.'&collection='.$collection.'&year='.$year.'&month='.$month.'">'.urldecode($collection).'</a> ['.$data['count'].']
```

PHPStan (2.2.2)

[error] Constant LINK not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(constant.notFound)